### PR TITLE
Bookstore: add reducers and actions

### PR DESCRIPTION
**Pull request by AxelSoler**
## Second part of the Bookstore project

- React Redux.
- Book's actions and reducer for adding and removing a book.
- Categories actions and reducer for checking the status.
- Configure the Redux Store

All feedback is well received.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+const booksReducer = (state = booksArray, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [
+        ...state,
+        action.book,
+      ];
+    case DELETE_BOOK:
+      return state.filter((book) => book.id !== action.book.id);
```
Comment (rindrajosia) [inline src/redux/books/books.js]: - [x] [REQUIRED] Please, pay attention on your parameters, if you check your deleteBook Action Creators, it returns `{type: DELETE_BOOK, id}` so here, `state.filter((book) => book.id !== action.book.id)` should be `state.filter((book) => book.id !== action.id)`

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! :muscle:

## To highlights 👍 

- Good job by creating book's actions and reducer
- Good job by creating categories actions and reducer
- Good job by configurirng the Redux Store

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- to use Redux middleware, like a redux logger to help you in the development process, to install: ``npm i --save redux-logger``

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 1 above)
Comment (AxelSoler) [reply src/redux/books/books.js]: Thank you, I will fix it now

**Review by rindrajosia — APPROVED** [review]
Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**